src/Depends/Program.cs

```diff
@@ -32,6 +32,8 @@ internal class Program
         [Option("--version <PACKAGEVERSION>", Description = "The version of the package to analyze.")]
         public string Version { get; }
 
+        [Option("-s|--solution", Description = "Solution mode, analyze solution rather than a project")]
```

Comment:
Instead of adding another option, what do you think of discovering `sln`, `csproj`, `vbproj`, `fsproj`, etc..? Here's one example for a project discovery used in `dotnet-outdated`. https://github.com/jerriep/dotnet-outdated/blob/master/src/DotNetOutdated.Core/Services/ProjectDiscoveryService.cs#L18

Reply:
It could make the tool more simple to use for sure. I don't think I could do it better than what's done in the example though, so I'll probably just use it with minor adaptations for dotnet-depends if that's OK.